connect-examples/v2/node_bookings/public/stylesheets/style.css

```diff
@@ -368,6 +384,10 @@ a:visited {
   width: 120px;
 }
 
+.availability button.available-time:hover {
```

Comment:
Would `.availability .available-time:hover` be sufficient, or is `button` needed for specificity reasons? If possible it would be good to avoid relying on the specific element type so we can be robust to future changes (like if this turned into an `<a>` tag or something in the future)